src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.Parser;
+import duke.dukeexception.DukeException;
+
+/**
+ * The Duke program is an interactive bot that offers commands to help the
+ * user keep track of a mutable list of tasks that can be of 3 types:
+ * <code>Todo</code>, <code>Deadline</code> or <code>Event</code>.
+ * These tasks can also be marked as done and will be saved in the hard disk.
+ *
+ * @author Foo Jing Yi
+ */
+public class Duke {
+    /** Storage object used by Duke to load from and write to hard disk */
```

Comment:
Perhaps you can move the description to the class file itself? To reduce clutter